src/main/java/br/com/edu/ifmt/sacode/application/configs/JwtTokenService.java

```diff
@@ -49,7 +52,7 @@ private Instant creationDate() {
     }
 
     private Instant expirationDate() {
-        return ZonedDateTime.now(ZoneId.of("America/Recife")).plusHours(4).toInstant();
+        return ZonedDateTime.now(ZoneId.of("America/Recife")).plusHours(12).toInstant();
```

Comment:
```suggestion
        return ZonedDateTime.now(ZoneId.of("America/Cuiaba")).plusHours(12).toInstant();
```

Verificar se é possível.